### PR TITLE
switch: fix ceph_uid fact

### PR DESCRIPTION
**Pull request by leseb**
Latest is now centos not ubuntu anymore so the condition was wrong.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by leseb** [comment]
`Failure talking to yum: Cannot find a valid baseurl for repo: base/7/x86_64`...

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
@leseb this needs to be backported otherwise nightlies jobs will fail

**Comment by guits** [comment]
@leseb I'm concerned regarding backward compatibility because of 203d48ab0
if someone has already deployed a cluster and defined pools in his infra with the 'old style' it will break anyway with a new run of the playbook. It might be an issue for OOO cases as well.

File: roles/ceph-client/tasks/create_users_keys.yml [hunk 1]
```diff
@@ -69,8 +69,8 @@
     {{ item.get('pg_num', hostvars[groups[mon_group_name][0]]['osd_pool_default_pg_num']) }}
     {{ item.pgp_num | default(item.pg_num) }}
     {{ item.rule_name | default("replicated_rule") }}
-    {{ item.type | default("replicated") }}
-    {%- if item.type | default("replicated") == 'erasure' and item.erasure_profile != '' %}
+    {{ item.type | default(1) }}
```
Comment (leseb) [inline roles/ceph-client/tasks/create_users_keys.yml]: what about this for backward compatibility?

```
{{ 1 if item.type|default(1) == 'replicated' else item.type|default(1) }}"
```

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-ooo_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-switch_to_containers

**Pull request merged**